File: 100-Days-Series/Day 58/FloydsTriangle.java
public class FloydsTriangle {
    // Function to print Floyds Triangle
    public static void getPrintFloydsTriangle(int rows){
        int num=1;
        for(int i=1;i<=rows;i++){
            for(int j=1;j<=i;j++){
                System.out.print((num++)+" ");
            }
            System.out.println();
        }
    }
    // Let's Start here
    public static void main(String [] args){
        int rows;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter the number of rows : ");
        rows=sc.nextInt();
        getPrintFloydsTriangle(rows);
    }

}

File: 100-Days-Series/Day 58/FloydsTriangle.py

# Function to print Floyd's Triangle
def getPrint_floyds_triangle(rows):
    num=1
    for i in range(0,rows):
        for j in range(0,i+1):
            print(num,end=" ")
            num+=1
        print()

# Let' Start here
rows=int(input('Enter the number of rows : '))
getPrint_floyds_triangle(rows)